sdk/tables/azure-tables/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# Release History
+
+## 1.0.0 (UNRELEASED)
```

Comment:
NIT: Consider changing the heading here to be `1.0.0-preview.1`, to align with the version of the first preview we'll ship.